packages/core/src/promotionEvaluations/redux/__tests__/selectors.test.js

```diff
@@ -0,0 +1,55 @@
+import * as fromPromotionEvaluations from '../reducer';
+import * as selectors from '../selectors';
+import {
+  mockErrorState,
+  mockLoadingState,
+  mockPromotionEvaluationItemId,
+  mockPromotionEvaluationsItemsResponse,
+  mockState,
+} from 'tests/__fixtures__/promotionEvaluations';
+
+describe('promotionEvaluations redux selectors', () => {
+  beforeEach(jest.clearAllMocks);
+
+  describe('arePromotionEvaluationItemsLoading()', () => {
+    it('should get the promotion evaluations items loading status', () => {
+      const spy = jest.spyOn(fromPromotionEvaluations, 'getIsLoading');
+
+      expect(
+        selectors.arePromotionEvaluationItemsLoading(mockLoadingState),
+      ).toBe(true);
```

Comment:
```suggestion
      ).toBe(mockLoadingState.isLoading);
```

I think it should get the actual value of the mock state, just like your other tests 🙂

Reply:
I agree. It had to be `mockLoadingState.promotionEvaluations.isLoading` though 🙂